bindings/jupyroot/python/JupyROOT/helpers/utils.py

```diff
@@ -319,7 +319,7 @@ def produceCanvasJson(canvas):
 
    ROOT.TColor.DefinedColors()
 
-   canvas_json = ROOT.TBufferJSON.ConvertToJSON(canvas, 3)
+   canvas_json = ROOT.TBufferJSON.ConvertToJSON(canvas, 23)
```

Comment:
I understand compression `23` is the maximum:
https://root.cern.ch/doc/master/classTBufferJSON.html#a38d20878d3af12bb03c4da37f4960ad3
I think the `3` is missing in the list for the second digit in the documentation of `ConvertToJSON`, though.

Perhaps we can add a comment here saying we apply maximum compression to the JSON?

Reply:
I will update docu - it should be 10 for "exclude leading and trailing zeros" and 20 for "searching and removing gaps"